package.json

```diff
@@ -342,6 +366,7 @@
 		"stylelint": "9.10.1",
 		"supertest": "3.4.2",
 		"svgstore-cli": "1.3.1",
+		"webpack": "4.29.6",
```

Comment:
See p1570092168081600-slack-team-calypso and https://github.com/WordPress/gutenberg/pull/17746